rllib/algorithms/cql/cql.py

```diff
@@ -68,8 +68,6 @@ def __init__(self, algo_class=None):
         self.min_q_weight = 5.0
 
         # Changes to Trainer's/SACConfig's default:
-        # .offline_data()
-        self.off_policy_estimation_methods = {}
```

Comment:
What will cqls default eval methods be now?

Reply:
Just None, same as before; I think if a user has access to a simulator for evaluation we should encourage `input=sampler` in the eval config...